static/app/views/performance/utils.tsx

```diff
@@ -71,8 +71,10 @@ export function platformToPerformanceType(
   }
 
   if (
-    selectedProjects.every(project =>
-      BACKEND_PLATFORMS.includes(project.platform as string)
+    selectedProjects.every(
+      project =>
+        BACKEND_PLATFORMS.includes(project.platform as string) &&
+        project.platform !== 'native'
```

Comment:
Which SDks do `native` include?

If the intent is to ensure `native` platforms go to All transactions, it'll be cleaner to ensure that `native` isn't in the `BACKEND_PLATFORMS` [here](https://github.com/getsentry/sentry/blob/c958b4bdc995f19c29b6e8e38723f42ceec7fadd/static/app/views/performance/utils.tsx#L48) with something like

```typescript
const BACKEND_PLATFORMS: string[] = backend.filter(platform => platform !== 'native');
```

Reply:
yes that would work too :)